### ISSUE TITLE
generating html

### ISSUE THREAD

**Issue by GavinHuttley** [QuantEcon/sphinxcontrib-jupyter.minimal#3]
For context, I'm investigating using `sphinxcontrib-jupyter` for documenting  a number of projects I have, [including this one](https://cogent3.readthedocs.io) which has interactive plotly plots.

My basic objective is to specify docs as rst, convert to notebooks, execute then translate into html.

I'm running into problems. For sphinxcontrib-jupyter.minimal, executing the makefile on the downloaded project works as advertised.

I have edited the Makefile using the command for producing html [from in the sphinxcontrib-jupyter docs](https://sphinxcontrib-jupyter.readthedocs.io/en/latest/builders.html).
```
# Minimal makefile for Sphinx documentation
#

# You can set these variables from the command line.
SPHINXOPTS    =
SPHINXBUILD   = python -msphinx
SPHINXPROJ    = sphinxcontrib-jupyterminimal
SOURCEDIR     = .
BUILDDIR      = _build

# Put it first so that "make" without argument is like "make help".
help:
	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)

.PHONY: help Makefile

# Catch-all target: route all unknown targets to Sphinx using the new
# "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
%: Makefile
	@$(SPHINXBUILD) -M jupyter "$(SOURCEDIR)" "$(BUILDWEBSITE)" $(FILES) $(SPHINXOPTS) $(O) -D jupyter_make_site=1 -D jupyter_generate_html=1 -D jupyter_download_nb=1 -D jupyter_execute_notebooks=1 -D jupyter_target_html=1 -D jupyter_download_nb_image_urlpath="https://s3-ap-southeast-2.amazonaws.com/lectures.quantecon.org/py/_static/" -D jupyter_images_markdown=0 -D jupyter_html_template="python-html.tpl" -D jupyter_download_nb_urlpath="https://lectures.quantecon.org/" -D jupyter_coverage_dir=$(BUILDCOVERAGE)
```
Executing it does not work. Suggestions appreciated!

**Comment by mmcky** [comment]
hi @GavinHuttley -- thanks for reporting this and your interest in using these tools.

Do you get an `executed` folder in your `_build/jupyter` folder?

My guess is you have incomplete compilation by looking at your `Makefile` because you don't have an `html` template that is needed when converting `ipynb` to `html`. Currently this is something that needs to be provided. 

https://sphinxcontrib-jupyter.readthedocs.io/en/latest/config-extension-html.html#jupyter-html-template

We are actually updating our `jupinx-quickstart` utility this week and we will be adding basic templates for `html` and `pdf` conversion to make this process much simpler which will help you setup a `jupinx` repository setup to work with scientific writing between `RST`, `IPYNB`, `HTML` and `PDF`. Writing a template is non-trivial. 

https://jupinx.quantecon.org

I will message you again once we have that release of `jupinx-quickstart` up and running this week. Is that OK?

**Comment by GavinHuttley** [comment]
Perfect, and thanks for prompt response!

In terms of getting an `_build/jupyter/executed` path, yes for the repo version of the Makefile. I hit errors when I edited.

**Comment by mmcky** [comment]
@GavinHuttley -- we have just released `jupinx==0.2.1` through PyPI which includes support of a `minimal` theme using `jupinx-quickstart`. It can be fetched from `pypi` using `pip install jupinx`.

This utility is similar to `sphinx-quickstart` but sets up a sphinx project that is compatible with `jupinx` tools and ensures all software is installed. This might be a quick way to get up and running.

http://jupinx.quantecon.org

https://github.com/QuantEcon/jupinx/releases
 
We would love any feedback you have or if you find something not working - please report.
